include/EASTL/algorithm.h

```diff
@@ -2670,10 +2670,16 @@ namespace eastl
 	remove(ForwardIterator first, ForwardIterator last, const T& value)
 	{
 		first = eastl::find(first, last, value);
-		if(first != last)
+		if(first != last) 
 		{
-			ForwardIterator i(first);
-			return eastl::remove_copy(++i, last, first, value);
+			for(ForwardIterator i = first; ++i != last;) 
```

Comment:
Please give this pattern a name like the code previously did.  ie.  "eastl::remove_copy"...  Perhaps eastl::remove_move?  I'm open to other/better names. :)